### DISCUSSION TITLE
I can't get the notifications to work

### DISCUSSION THREAD

**Discussion by stamateas** [upptime/upptime#220] [Q&A] [answered]
Maybe I am doing something wrong and someone can help me. 

I think I setup everything as described https://upptime.js.org/docs/notifications but still I cant get it to work. I setup both Telegram and Slack notifications but still not receiving anything. 

What I did is: 

- Went to Setting --> Secrets --> New Repository Secret 
- Added 1st Secret with Name: NOTIFICATION_SLACK and Value: true
- Added 2nd Secret with Name: NOTIFICATION_SLACK_WEBHOOK and Value: true
- Added 3rd Secret with Name: NOTIFICATION_SLACK_WEBHOOK_URL and Value; https://WEBHOOURL-GENERATED-BY-SLACK
- Did this similar for Telegram as discribed in the docs. 
- Updated .upptimerc.yml
- Took off my webpage and waited for a notification but no notification

Anyone an idea what I might be doing wrong?

Thanks

**Comment by AnandChowdhary** [comment, marked as answer]
This is now (finally) fixed, thanks so much for your patience! The fix is available in [`upptime/uptime-monitor@v1.26.2`](https://github.com/upptime/uptime-monitor/releases/tag/v1.26.2), and the automated fix should be in your repositories within 24 hours.

You can also run Update Template CI [manually](https://upptime.js.org/docs/triggers) to update to the latest version immediately; by default this runs once every day.
